tests/integration/test_ols.py

```diff
@@ -14,7 +14,7 @@ def test_healthz() -> None:
 
 
 def test_readyz() -> None:
-    response = client.get("/healthz")
```

Comment:
But we support both in app.py - shouldn't we test both? This just changes which one we test.

Reply:
we do test both. /healtz is tested in `test_healthz` just few lines above. Basically these two functions call the one API